decidim-core/app/services/decidim/truncation.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module Decidim
+  class Truncation
```

Comment:
I think we should elaborate this is for HTML truncation, not for truncating normal text.